testsuite/features/profiles/cloud_aws/Kiwi/POS_Image-JeOS7_head/images.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Copyright (c) 2019-2020 SUSE LLC
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+test -f /.kconfig && . /.kconfig
+test -f /.profile && . /.profile
+
+systemctl enable salt-minion.service
```

Comment:
A good question for a different PR :-)

Reply:
@aaannz @meaksh ? ^^^

Reply:
`images.sh` is called during image building process - https://osinside.github.io/kiwi/overview/workflow.html#description-components

Changing to salt bundle is not as simple as changing this call, but we pushed salt bundle capable initrd in 4.3.2. Changed templates are not yet published, waiting until release. See https://github.com/SUSE/manager-build-profiles/pull/33 for changes.

It would be great if we figure out some way to pass repos in different way then editing XML so you would be able to run very same templates our users use.

Reply:
Has this something to be with the comment from @nadvornik here?
https://github.com/SUSE/spacewalk/issues/18970#issuecomment-1252044816

Reply:
Rather than editing the XML at run time, you might also use a different profile for 4.3 than for 4.2. It might be simpler. I susepect other divergences will appear in the future.